src/App.js

```diff
@@ -1,27 +1,11 @@
-import logo from './logo.svg';
+import 'bootstrap/dist/css/bootstrap.min.css';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Good job on the design however if you can style it exactly as the one below it would be great

<img width="558" alt="calculator" src="https://user-images.githubusercontent.com/25227022/179048144-72e62e58-477c-4824-8545-e37b6508187f.png">